.travis.yml

```diff
@@ -53,10 +53,12 @@ before_script:
 - node --version
 - rm -rf ./node_modules package-lock.json
 - travis_retry npm install
+- if [ "$PROM_METRICS_TEST" = "1" ]; then npm install prom-client@11.0.0; fi
 
 script:
 - if [ "$LINT" = "1" ]; then npm run lint; fi
 - if [ "$CROSSDOCK" != "1" ]; then make test-without-build; fi
+- if [ "$PROM_METRICS_TEST" = "1" ]; then npm run test-prom-metrics; fi
```

Comment:
I think it's better to have these as part of regular test, but skip them when env var is not set. This is how we are doing tests against Cassandra / Elastic / Redis in other Jaeger repos.

Reply:
> I think it's better to have these as part of regular test, but skip them when env var is not set.

What I understand from this and below comment(from test/metrics/prometheus.js's )
- Run all tests including `test-prom-metrics` by default.
- Run all tests except `test-prom-metrics` if env is present (or indicate it's 0.10) 

Is this ok? Pls correct me if I misunderstand.

> This is how we are doing tests against Cassandra / Elastic / Redis in other Jaeger repos.

Could you show me the location for reference ?

Reply:
Here's the example: https://github.com/jaegertracing/jaeger/blob/2fb6fa5d1928278ccaf164064966d05e18a9f336/plugin/storage/integration/elasticsearch_test.go#L109